docs/BUILD

```diff
@@ -82,6 +82,14 @@ tarball(
     labels = ["hlink:plz-out/pkg"],
 )
 
+tarball(
```

Comment:
The flat tarball was harder to deal with and was flattening the images into the web root for some reason.